week-1/Homework/mandatory/4-tax.js

```diff
@@ -17,7 +19,11 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function formatCurrency() {}
+function formatCurrency(sales) {
+  var sales1 = sales * 1.2;
+  console.log("£" + sales1.toFixed(2));
+  return "£" + sales1.toFixed(2);
+}
```

Comment:
It's ok here the only thing you missed is the function above is doing the calculation for you. You just need to use it and add the currency symbol.